Products/Jobber/model.py

```diff
@@ -193,6 +193,7 @@ class JobRecordMarshaller(object):
         "finished",
         "status",
         "userid",
+        "uuid",
```

Comment:
You'll need to update some unit tests to expect the extra field.

Reply:
Fixed unit test related to this case